### PR TITLE
Update for each iEta region plotting

### PR DESCRIPTION
**Pull request by gechen**
@jsturdy

### CODE REVIEW

File: MaskChannels.py [hunk 1]
```diff
@@ -6,15 +6,14 @@
 @author: Geng
 
 """
-
-from ROOT import gROOT, gDirectory, TMultiGraph, TNamed, TLegend, TCanvas, TGraph, TH1F, TH2F, TFile, TDirectory #Classes
-from ROOT import kGreen, kYellow, kBlue, kRed #Colors
+from ROOT import *
```
Comment (jsturdy) [inline MaskChannels.py:9]: Don't need both line 9 and 10.
Probably it's simplest to remove line 10, and change line 11 to be:
```gROOT.SetBatch(True)```

**Review by jsturdy — CHANGES_REQUESTED** [review]
Can you make the change I suggest?
Then I'll approve and merge

File: MaskChannels.py (hunk 1 above)
Comment (gechen) [reply MaskChannels.py:9]: Updated.

**Review by jsturdy — APPROVED** [review]
Thanks!

**Pull request merged**